src/Components/SmartComponents/SystemCves/SystemCves.js

```diff
@@ -213,12 +213,25 @@ class SystemCves extends Component {
     };
 
     showStatusModal = cves => {
+
+        let hasDifferentStatus = false;
+        if (cves.length > 1) {
+            const selected_cves = Array.from(this.props.cveList.data.filter(function (cve) {
```

Comment:
Interesting how lint didn't complain about the `selected_cves` variable.  It should be camelCase based on the rule https://github.com/RedHatInsights/vulnerability-ui/blob/master/.eslintrc.yml#L62

Reply:
Oh, I see. on top of the file, we have `/* eslint-disable camelcase */` which disables errors in the file for this specific rule. Anyway, could you please change it to camelCase? :nerd_face: